reactive/etcd.py

```diff
@@ -155,30 +140,28 @@ def leader_config_changed():
             # Update the member's peer_urls with the new ports.
             log(etcdctl.member_update(members[unit_name]['unit_id'], url))
         # Render just the leaders configuration with the new values.
-        render('defaults', '/etc/default/etcd', bag.__dict__, owner='root',
-               group='root')
+        render_config()
```

Comment:
Passing in Bag is only really necessary when you have changes to the bag object (like updating a cluster string with a seeded value from leader_data, as seen in line: 315. 

Good call out, but I think this is an acceptable time to use the objects probed defaults.